pinot-broker/src/main/java/com/linkedin/pinot/broker/routing/builder/LowLevelConsumerRoutingTableBuilder.java

```diff
@@ -81,32 +81,32 @@ protected RoutingTableGenerator buildRoutingTableGenerator() {
 
     private Map<String, List<String>> _segmentToServersMap = new HashMap<>();
 
-    public KafkaLowLevelConsumerRoutingTableGenerator() {
+    public LowLevelConsumerRoutingTableGenerator() {
       super(_targetNumServersPerQuery);
     }
 
     @Override
     public void init(ExternalView externalView, List<InstanceConfig> instanceConfigList) {
-      // 1. Gather all segments and group them by Kafka partition, sorted by sequence number
-      Map<String, SortedSet<SegmentName>> sortedSegmentsByKafkaPartition =
+      // 1. Gather all segments and group them by partition, sorted by sequence number
+      Map<String, SortedSet<SegmentName>> sortedSegmentsByPartition =
```

Comment:
may be you name this variable sortedSegmentsByStreamPartition. Since there is also the concept of partition based routing, this may be less confusing.